src/verifiers/issuerIdentity/didIdentityProof/didIdentityProof.test.ts

```diff
@@ -0,0 +1,33 @@
+import { OpenAttestationDidSignedDidIdentityProof } from "./didIdentityProof";
+
+describe("skip", () => {
+  it("should return skip message", () => {});
+});
+
+describe("test", () => {
```

Comment:
I would suggest to follow the structure of other test and to not test separately for test/verify, rather call it through a build in verifier (because that's how verifier are used)

Reply:
hmmm i found the structure doing a bit too many things at once, perhaps can have an test around it?

Reply:
the problem by separating test / verify is that there is an implicit dependency: when there is a test in the test section returning false, you don't expect the verify section to run the same test because it shouldn't happen.

However if at some point the need change and the test that was previously returning false, returns true, then you would expect a new test to be created in the verify section which can be easily missed.

I don't understand when you say ` i found the structure doing a bit too many things at once`, iirc tests are only running one verifier and checking the fragment ?